deployments/Dockerfile

```
@@ -1,22 +1,20 @@
-# This Dockerfile is used to build the image available on DockerHub
-FROM centos:centos7 as build
+FROM alpine:3.19 as build
```

```suggestion
FROM alpine:3.20 as build
```
Why not Alpine 3.20? Did you face any problems with that?